llama-cli/src/main.rs

```diff
@@ -2,7 +2,7 @@ use std::{convert::Infallible, io::Write};
 
 use cli_args::CLI_ARGS;
 use llama_rs::{InferenceParameters, InferenceSnapshot};
-use rand::thread_rng;
+use rand::{thread_rng, SeedableRng};
```

Comment:
`thread_rng` is no longer used

Reply:
Looks like CI's borked, it didn't catch this: https://github.com/setzer22/llama-rs/actions/runs/4455272439/jobs/7824987492?pr=36

Clippy emitted the warning, but the test didn't faill

Reply:
Looks like we need to add `-Dwarnings` to the clippy command